models/experimental/functional_stable_diffusion/tt2/ttnn_functional_unet_2d_condition_model.py

```diff
@@ -580,18 +580,7 @@ def __call__(
                 memory_config=self.gn_expected_input_sharded_memory_config,
                 core_grid=self.group_norm_core_grid,
             )
-        sample = ttnn.to_memory_config(sample, ttnn.L1_MEMORY_CONFIG)
-        sample = ttnn.reshape(
-            sample,
-            (
```

Comment:
Similar as above, what's the input shape here, the conv expects 1,1,bhw,c

Reply:
right, it is (b, 1, hw, c). I'll add the reshape back.

Reply:
I'm a little baffled that the test passes without it... did you see any PCC drop?

Reply:
no I didn't notice obvious drop, the unet test passes.